.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: ['1.6', '1.7', '^1.8.0-beta3']
+        version: ['1.6', '1.7', '1.8', '^1.9.0-beta3']
```

Comment:
rc1?

Reply:
Just got tagged I don't think we have binaries yet.